.github/workflows/ci-cd.yml

```
@@ -30,8 +30,97 @@ jobs:
+          # Start the backend service with nohup to ensure it stays running
+          echo "Starting backend service..."
+          nohup pnpm run dev:backend > backend.log 2>&1 &
+          # Wait for the backend to initialize
+          sleep 15
+
+      - name: Wait for backend with OK status
+        run: |
+          echo "Checking if backend is running with OK status..."
+          max_attempts=12
+          attempts=0
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)